site/src/pages/DeploySettingsPage/AuthSettingsPage.tsx

```diff
@@ -42,7 +42,6 @@ const AuthSettingsPage: React.FC = () => {
           <OptionsTable
             options={{
               oidc_client_id: deploymentConfig.oidc_client_id,
-              oidc_client_secret: deploymentConfig.oidc_client_secret,
```

Comment:
We can add this back once I get better sensitive data support. I already have a branch I'm working on for it.

Reply:
neato